src-plugins/itkFilters/itkFiltersAddProcess.cpp

```diff
@@ -70,63 +70,60 @@ int itkFiltersAddProcess::update ( void )
 {
     DTK_D(itkFiltersAddProcess);
     
-    if ( !d->input )
-        return -1;
-
-    QString id = d->input->identifier();
-
-    qDebug() << "itkFilters, update : " << id;
-
-    if ( id == "itkDataImageChar3" )
-    {
-        d->update<char>();
-    }
-    else if ( id == "itkDataImageUChar3" )
-    {
-        d->update<unsigned char>();
-    }
-    else if ( id == "itkDataImageShort3" )
-    {
-        d->update<short>();
-    }
-    else if ( id == "itkDataImageUShort3" )
-    {
-        d->update<unsigned short>();
-    }
-    else if ( id == "itkDataImageInt3" )
-    {
-        d->update<int>();
-    }
-    else if ( id == "itkDataImageUInt3" )
-    {
-        d->update<unsigned int>();
-    }
-    else if ( id == "itkDataImageLong3" )
-    {
-        d->update<long>();
-    }
-    else if ( id== "itkDataImageULong3" )
-    {
-        d->update<unsigned long>();
-    }
-    else if ( id == "itkDataImageFloat3" )
-    {
-        d->update<float>();
-    }
-    else if ( id == "itkDataImageDouble3" )
-    {
-        d->update<double>();
-    }
-    else
+    if ( d->input )
     {
-        qDebug() << description()
-                 <<", Error : pixel type not yet implemented ("
-                 << id
-                 << ")";
-        return -1;
+        QString id = d->input->identifier();
+
+        if ( id == "itkDataImageChar3" )
+        {
+            return d->update<char>();
```

Comment:
(by the way, it's done @LoicCadour. The Conversation tab behavior is strange since the last version of github)